docs/integrate.md

```diff
@@ -1,49 +1,70 @@
+## 1: Choose your UI platform
 
-![Architecture](img/image1.png)<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-**_High-level representation of user’s data flow between systems_**
-
-Now, we will show you the steps to follow on how to integrate your application with Ultra platform. If you face any issues or queries while performing integration, please reach out to [us](contact.md).
-
-## Step 1: Choose your UI platform
-
-You can build your application on Ultra using either [React Native](https://facebook.github.io/react-native/) platform or simple HTML /PWA. Ultra supports both containers and we can help you in making this choice by spotlighting a comparison between these two platforms as below:
+Ultra supports [React Native](https://facebook.github.io/react-native/) as well as a simple HTML /PWA. You can build your application using either of these and we can help you in making the best choice by spotlighting a comparison between the two as follows:
 
 | **React Native** | **HTML** |
 |--------------|------|
 | Supports JavaScript code | Supports JavaScript code |
 | Extraordinary performance | Satisfactory performance |
 | Highly optimised for mobile applications | Average optimisation |
-| Bundle is delivered from Flipkart CDN, only differential components are downloaded and is cached within the Flipkart application | Bundle and assets follow cache-control headers and might be slower to render even from cache |
+| Bundle is delivered from Flipkart CDN, only differential components are downloaded and is cached | Bundle and assets follow cache-control headers and might be slower to render even from cache |
 
 ### Ultra’s interaction with UI platform
 
 #### Using React Native
 
-If you choose React Native, Ultra will pull in your React Native bundle from your GitHub repository and then deliver to Flipkart application over-the-air. Flipkart receives these React Native JavaScript bundles via Dynamic Update Service ([DUS](https://github.com/Flipkart/DUS)). We recommend you to visit [this link](https://github.com/Flipkart/DUS) to understand how this works.
+Ultra pulls in the React Native bundles from your GitHub repository and then delivers it to Flipkart dynamically which we prefer to call as “Over-The-Air”. Flipkart receives these React Native JavaScript bundles via Dynamic Update Service (DUS). We recommend you to visit [this link](https://github.com/Flipkart/DUS) to understand how DUS works.
 
-Once DUS fetches your bundle within the Flipkart application, it will load your React Native experience within a React Fragment from where you can navigate to the other pages.
+Once DUS fetches your bundle within the Flipkart, it loads the React Native application within a React Fragment from where you can navigate to the other pages.
```

Comment:
word "application" is missing after "within the Flipkart". The first part of the sentence should read as "Once DUS fetches your bundle within the Flipkart application,"....